File: control-assurance-sp/src/types/NAOAssignment.ts
import { IEntity } from "./Entity";

export interface INAOAssignment extends IEntity {
    NAORecommendationId?: number;
    UserId?: number;
}

export class NAOAssignment implements INAOAssignment {
    public ID = 0;
    public Title = null;
    public NAORecommendationId = null;
    public UserId = null;
}

File: ControlAssuranceAPI/Models/NAOAssignment.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class NAOAssignment
{
    public int ID { get; set; }

    public string? Title { get; set; }

    public int? NAORecommendationId { get; set; }

    public int? UserId { get; set; }

    public DateTime? DateAssigned { get; set; }

    public virtual NAORecommendation? NAORecommendation { get; set; }

    public virtual User? User { get; set; }
}
